content/src/main/content/jcr_root/apps/acs-commons/components/content/sitewide-component-props-example/.content.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@davidjgonzalez Looking at the ACS Samples project, wondering the best way to incorporate this.  There's a site at /content/acs-samples but it contains a (currently broken on 6.1) sample angularJS site called "AngularJS App" directly below it, with /content/acs-samples redirecting to it at /content/acs-samples/angularjs-app.

Would it make sense for me to add sample pages to the same angularjs-app website that demonstrate this new feature?  Or should I create a new site at /content/acs-samples/acs-shared-props?  Want to make sure I'm in line with the vision for that project.